native/src/components/SubCategoryList.tsx

```diff
@@ -52,35 +52,35 @@ export const CategoryThumbnail = styled(SimpleImage)<{ language: string }>`
     isContentDirectionReversalRequired(props.language) ? dimensions.categoryListItem.margin : 0}px;
 `
 
-type CategoryListItemProps = {
+type SubCategoryListProps = {
   category: CategoryModel
   subCategories: CategoryModel[]
   resourceCache: LanguageResourceCacheStateType
   onItemPress: (item: { path: string }) => void
   language: string
 }
 
-const CategoryList = ({
+const SubCategoryList = ({
   language,
   category,
   subCategories,
   resourceCache,
   onItemPress,
-}: CategoryListItemProps): ReactElement => (
+}: SubCategoryListProps): ReactElement => (
   <>
     <FlexStyledLink onPress={() => onItemPress({ path: category.path })}>
       <DirectionContainer language={language}>
-        <CategoryEntryContainer>
+        <SubCategoryEntryContainer>
```

Comment:
Sorry but these renamings are wrong :see_no_evil: The sub categories are only rendered below (lines 86 until 98). If I think more about it, the name of this should actually stay `CategoryListItem`. The component renders a category list item. The fact that a part of that is that it renders sub categories is just an implementation detail and not really relevant to the usage... Sorry for the confusion, my mistake!